packages/frontend/src/components/accounts/ledger/SignInLedgerViews/EnterAccountId.js

```diff
@@ -26,15 +31,15 @@ const EnterAccountId = ({
                 mainLoader={mainLoader}
                 handleChange={handleChange}
                 checkAvailability={checkAccountAvailable}
-                localAlert={localAlert}
+                localAlert={status.localAlert}
```

Comment:
```suggestion
                localAlert={status?.localAlert}
```
`status` has optional chaining below, do we need it here too or will `status` always be defined?

Reply:
`status` and `localAlert` prop are always set by the initial state.